tradable-app/src/components/App/App.js

```diff
@@ -0,0 +1,31 @@
+import React, { useContext, useEffect } from "react";
+import Context from "../../store/Context";
+import SideMenu from "../SideNav/SideNav";
+import TopMenu from "../TopMenu/TopMenu";
+import Dashboard from "../../pages/Dashboard/Dashboard";
+import { Container, Main } from "./style";
+import io from "socket.io-client";
+const ENDPOINT = "http://localhost:5000/";
```

Comment:
move it to an .ENV file and add it to npm ignore as we don't want to show private things as mongo keys to others